src/test/java/bio/terra/janitor/app/controller/JanitorApiControllerTest.java

```diff
@@ -100,16 +100,16 @@ private void assertResourceExists(CreatedResource createdResource) {
   }
 
   private static String newJsonCreateRequestBody(
-      CloudResourceUid cloudResourceUid, Optional<Map<String, String>> labels) {
-    ObjectMapper mapper = new ObjectMapper();
-
-    ObjectNode trackedResourceNode =
-        mapper.createObjectNode().put("timeToLiveInMinutes", TIME_TO_LIVE_MINUTE);
-    trackedResourceNode.set("resourceUid", mapper.valueToTree(cloudResourceUid));
+      CloudResourceUid cloudResourceUid, Optional<Map<String, String>> labels)
+      throws JsonProcessingException {
+    CreateResourceRequestBody body =
+        new CreateResourceRequestBody()
+            .resourceUid(cloudResourceUid)
+            .timeToLiveInMinutes(TIME_TO_LIVE_MINUTE);
     labels.ifPresent(
         l -> {
-          trackedResourceNode.set("labels", mapper.valueToTree(l));
+          body.setLabels(l);
         });
-    return trackedResourceNode.toString();
+    return new ObjectMapper().writeValueAsString(body);
```

Comment:
I don't think we need this method anymore. It's simple enough to create the CreateResourceRequestBody directly and serialize it.
Factory methods for objects that have a builder are an anti-pattern. If we want to have a DEFAULT_CREATE_REQUEST with some of the fields already initialized, that's a better testing pattern.

Reply:
deleted